website/docs/OrganizingTestSuites.md

```diff
@@ -166,6 +166,9 @@ Note that each test file is running in a single test runner process. Since we do
 
 This feature will help you to accomplish the same goal.
 
+When the `--spec` option is provided, it will override any patterns defined by the config or capability level's `specs`
+parameter.
```

Comment:
```suggestion
When the `--spec` option is provided, it will override any patterns defined by the config or capability level's `specs` parameter.
```